### PR TITLE
build script for package codemirror ubi8.4

### PR DESCRIPTION
**Pull request by vikasgupta8**
build script for package codemirror ubi8.4

### CODE REVIEW

File: c/classmate/classmate_8.4.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
+#----------------------------------------------------------------------------
+#
+# Package		: classmate
+# Version		: 1.3.4, 1.5.1
+# Source repo		: https://github.com/FasterXML/java-classmate
+# Tested on		: RHEL 8.4
+# Script License	: Apache License Version 2.0
+# Maintainer		: Vikas Gupta <vikas.gupta8@ibm.com>
```
Comment (kandarpamalipeddi) [inline c/classmate/classmate_8.4.sh:8]: Please update the header as per the templates.

File: c/classmate/classmate_8.4.sh [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+LOGS_DIRECTORY=/logs
+LOCAL_DIRECTORY=/home/tester
+REPOSITORY="https://github.com/FasterXML/java-classmate.git"
+
+
+# ------- Clone and build source -------
+
+export PATH=/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.312.b07-2.el8_5.ppc64le/bin/:$PATH
+java -version
+yum install maven
```
Comment (kandarpamalipeddi) [inline c/classmate/classmate_8.4.sh:36]: Move this installation before updating the PATH variables and running the `java`.

File: c/classmate/classmate_8.4.sh [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
```
Comment (kandarpamalipeddi) [inline c/classmate/classmate_8.4.sh:18]: Use `#!/bin/bash -e`

File: c/codemirror/codemirror_ubi8.4.sh [hunk 4]
```diff
@@ -0,0 +1,50 @@
+# ----------------------------------------------------------------------------
+#
+# Package	: CodeMirror
+# Version	: 5.26.0
+# Source repo	: https://github.com/codemirror/CodeMirror
+# Tested on	: rhel_8.4
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Vikas Gupta <vikas.gupta8@ibm.com>
```
Comment (kandarpamalipeddi) [inline c/codemirror/codemirror_ubi8.4.sh:8]: Please update the header as per the templates.

File: c/codemirror/codemirror_ubi8.4.sh [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+#
+# Disclaimer: This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
```
Comment (kandarpamalipeddi) [inline c/codemirror/codemirror_ubi8.4.sh:17]: Use `#!/bin/bash`

**Review by kandarpamalipeddi — CHANGES_REQUESTED** [review]

File: c/classmate/classmate_8.4.sh (hunk 1 above)
Comment (vikasgupta8) [reply c/classmate/classmate_8.4.sh:11]: Updated latest template.

File: c/codemirror/codemirror_ubi8.4.sh (hunk 5 above)
Comment (vikasgupta8) [reply c/codemirror/codemirror_ubi8.4.sh]: Added.

File: c/classmate/classmate_8.4.sh (hunk 3 above)
Comment (vikasgupta8) [reply c/classmate/classmate_8.4.sh]: added.

File: c/classmate/classmate_8.4.sh (hunk 2 above)
Comment (vikasgupta8) [reply c/classmate/classmate_8.4.sh]: Done.

**Review by kandarpamalipeddi — APPROVED** [review]

**Review by kandarpamalipeddi — CHANGES_REQUESTED** [review]
Please fix the merge conflicts.

**Comment by vikasgupta8** [comment]
Resolved conflicts after removing classmate_8.4.sh file.

**Comment by seth-priya** [comment]
@vikasgupta8 looks like the travis build is failing?

**Comment by vikasgupta8** [comment]
>

**Comment by vikasgupta8** [comment]
Build and test are passing successfully in empty container. Not sure why Travis build is failing.

_> codemirror@5.26.0 prepublish
> npm run-script build


> codemirror@5.26.0 build
> rollup -c


added 119 packages, and audited 120 packages in 11s

1 package is looking for funding
  run `npm fund` for details

4 vulnerabilities (1 moderate, 3 high)

To address all issues possible (including breaking changes), run:
  npm audit fix --force

Some issues need review, and may require choosing
a different dependency.

Run `npm audit` for details.

> codemirror@5.26.0 test
> node ./test/run.js


All passed

LEAK: 118 Range
LEAK: 91 CachedResource
LEAK: 5816 WebCoreNode_

**Comment by seth-priya** [comment]
This has been added to currency as part of https://github.com/ppc64le/build-scripts/pull/3481 , hence closing

**Pull request closed without merging**